pyanaconda/ui/tui/spokes/language_support.py

```diff
@@ -111,9 +104,9 @@ def refresh(self, args=None):
                 self._container.add(widget, self._set_locales_callback, locale)
         else:
             self.window.add(TextWidget(_("Available languages")))
-            for lang in self._langs:
-                langs_and_locales = self._langs_and_locales[lang]
-                locales = self._locales[langs_and_locales]
+            available_translations = self._l12_module.GetLanguages()
+            for lang in available_translations:
+                locales = (lang, localization.get_language_locales(lang))
```

Comment:
I think it should be just `localization.get_language_locales(lang)`. This version will probably crash. Feel free to avoid any changes in the text mode since they are not critical parts of the pull request.

Reply:
Removed text mode changes